test_container/tests/lang/java/timezone.sql

```diff
@@ -0,0 +1,30 @@
+
+CREATE OR REPLACE java SCALAR SCRIPT
+DEFAULT_TZ()
+RETURNS VARCHAR(100) AS
+import java.time.ZoneId;
+import java.util.TimeZone;
+
+class DEFAULT_TZ{
+    static String run(ExaMetadata exa, ExaIterator ctx) throws Exception {
+        TimeZone timeZone = TimeZone.getDefault();
+        return timeZone.getDisplayName(false, TimeZone.SHORT);
+
+    }
+}
+/
+
+CREATE OR REPLACE java SCALAR SCRIPT
+MODIFY_TZ_TO_NEW_YORK()
+RETURNS VARCHAR(100) AS
+%jvmoption -Duser.timezone=America/New_York;
```

Comment:
Should we maybe also test, what happens when TZ is overwritten by %env

Reply:
Hm, I tried again using `%env` and it works....using this approach now

Reply:
Done